### PR TITLE
Convert icons to SVG

### PR DESCRIPTION
**Pull request by lukecyca**
The PNG icons were a bit blurry on modern high-DPI mobile and laptop screens, so I decided to swap them out for SVG versions which are based on Material UI, and then hand-edited. These look super sharp, and have slightly smaller file sizes.

IMG_0552

### CODE REVIEW

**Comment by tonyp7** [comment]
Hey Lucas this is amazing!

Could you please specify what is the license of these icons though? I cannot merge this if these icons are proprietary.

Thank you!

**Comment by lukecyca** [comment]
Absolutely! From https://github.com/google/material-design-icons:

> We have made these icons available for you to incorporate into your products under the Apache License Version 2.0. Feel free to remix and re-share these icons and documentation in your products. We'd love attribution in your app's about screen, but it's not required. The only thing we ask is that you not re-sell these icons.

**Review by tonyp7 — APPROVED** [review]
LGTM!

**Pull request merged**
